common/akv.bicep

```diff
@@ -0,0 +1,26 @@
+param keyVaultName string
```

Comment:
I would recommend that moving `common/akv.bicep` and `akv` into the fold `disk-encryption/bring-your-own-keys-for-disks`.

In future, if we find another sample needs the same bicep, then we can make them in common.

Reply:
fixed